Mandatory `metric` dimension (#896)
SumoLogic/sumologic-documentation: docs/metrics/introduction/metric-formats.md

Graphite does not support meta tags. However, you can use Sumo's metric rules editor to tag metrics with key-value pairs derived from a Graphite metric’s metric_path. Then, you can use those key-value pairs in metric queries. For more information, see [About Metric Rules](docs/metrics/metric-rules-editor#about-metric-rules).
:::

### Inferred metric name

> Due to its nature, Grahpite format doesn't have a notion of named tags. Since Sumo Logic requires each metric to have a valid metric name, in case of Graphite it has to be inferred in some way. Therefore, for every graphite metric sent to our backend, Sumo Logic will add additional `metric` tag with value equal to the last segment of a dot-separated `metric_path`.

Reviewer: ```suggestion
The Graphite format doesn't have a notion of named tags. Since Sumo Logic requires each metric to have a valid metric name, for Graphite metrics, the metric name must be inferred. To accomplish that, for every Graphite metric sent to our backend, Sumo Logic adds an additional `metric` tag, whose value is equal to the last segment of a dot-separated `metric_path`.
```

For example, for below graphite metric sent to Sumo Logic:
```
cluster-1.node-1.cpu-1.cpu-idle  73.12 1112470620
```
the inferred metric name will be `metric=cpu-idle` and such a key-value pair will be added

## Carbon 2.0

Carbon 2.0 metrics conform to the [Metrics 2.0](http://metrics20.org/) specification.

Carbon 2.0 metrics look like this:

```

> intrinsic_tags  meta_tags value timestamp

Reviewer 2: ```suggestion
intrinsic_tags meta_tags value timestamp
```
